packages/api/package.json

```diff
@@ -19,8 +19,10 @@
         "doc": "npx compodoc --tsconfig ../../tsconfig.json --hideGenerator --theme readthedocs --includes compodoc --output ../../docs/api"
     },
     "dependencies": {
+        "@cennznet/cennznut": "^1.0.0",
```

Comment:
I believe @cennznet/cennznut and @plugnet/doughnut-maker are just devDependencies not dependencies.